### PR TITLE
refactor(core): ARC-69 ignore rest of sonar smells

### PR DESCRIPTION
**Pull request by sf-sahil-jassal**
ignore sonar code smells related to error handling consoles and unpredictable any types

### CODE REVIEW

File: src/redux/apiSlice.ts [hunk 1]
```diff
@@ -34,7 +34,8 @@ const baseQueryWithReauth: BaseQueryFn<string | FetchArgs, unknown, FetchBaseQue
 
   if (result.error && result.error.status === RESULT_ERROR_STATUS) {
     // try to get a new token
-    console.log('sending refresh token....');
+    // eslint-disable-next-line no-console
+    console.log('sending refresh token....'); // NOSONAR
```
Comment (yeshamavani) [inline src/redux/apiSlice.ts:38]: are these consoles really necessary

File: src/Helpers/authorizationFunctions.ts [hunk 2]
```diff
@@ -1,14 +1,15 @@
 import useAuth from 'Hooks/useAuth';
 
+export interface AnyFunctions {
+  [key: string]: (...args: any[]) => boolean; // NOSONAR
+}
+
 const IsAuthenticated = () => {
+  // NOSONAR typescript:S100
```
Comment (yeshamavani) [inline src/Helpers/authorizationFunctions.ts:8]: ??

File: configGenerator.js [hunk 3]
```diff
@@ -51,12 +51,14 @@ try {
...
-    console.log('Configuration file written in current directory.');
+    // eslint-disable-next-line no-console
+    console.log('Configuration file written in current directory.'); // NOSONAR
 
     // Copy config.json to the public directory
     try {
       fs.copyFileSync(currentConfigFilePath, outConfigFilePath);
-      console.log('Configuration file copied to public directory.');
+      // eslint-disable-next-line no-console
+      console.log('Configuration file copied to public directory.'); // NOSONAR
```
Comment (yeshamavani) [inline configGenerator.js:61]: same is the necessary

File: configGenerator.js [hunk 4]
```diff
@@ -51,12 +51,14 @@ try {
   // Write the substituted data to config.json
   try {
     fs.writeFileSync(currentConfigFilePath, substitutedData, 'utf-8');
-    console.log('Configuration file written in current directory.');
+    // eslint-disable-next-line no-console
+    console.log('Configuration file written in current directory.'); // NOSONAR
```
Comment (yeshamavani) [inline configGenerator.js:55]: same is the necessary

**Review by yeshamavani — CHANGES_REQUESTED** [review]

File: src/redux/apiSlice.ts (hunk 1 above)
Comment (Yogi-Vishwas) [reply src/redux/apiSlice.ts:38]: This is not needed. This can be removed.

File: configGenerator.js (hunk 3 above)
Comment (Yogi-Vishwas) [reply configGenerator.js:61]: This is only during the development, we can keep this.

File: configGenerator.js (hunk 4 above)
Comment (Yogi-Vishwas) [reply configGenerator.js:55]: Same for this this is only during development.

File: src/Helpers/authorizationFunctions.ts (hunk 2 above)
Comment (sf-sahil-jassal) [reply src/Helpers/authorizationFunctions.ts:8]: According to React, name of function should be in PascalCase.
According to Sonar, name should be camelCase. I ignored sonar one.
I mentioned sonar rule as further

**Pull request merged**